### ISSUE TITLE
The title bar shows image selected count after pressing back

### ISSUE THREAD

**Issue by Swati4star** [Swati4star/Images-to-PDF#571]
### steps
- go to view files section
- select few files by selecting the checkbox
- press back

### actual behaviour
The title bar shows the selected images count
<img width="355" alt="screenshot 2019-02-26 at 10 50 23 pm" src="https://user-images.githubusercontent.com/9164477/53432680-f70d9780-3a18-11e9-9ebe-9a72a13b9fd8.png">

### expected behaviour
It should show app name : "PDF Converter"

**Comment by jddeep** [comment]
@Swati4star I have taken up one issue earlier and shall create a PR for the same on 1st March. Can this be my 2nd issue? I would like to work on this. Kindly confirm and assign.

**Comment by Swati4star** [comment]
Hi, you cant be assigned 2 issues at a time. You can be assigned second issue, only after your first Pr is approved.

**Comment by jddeep** [comment]
@Swati4star okay i shall make a PR for my first issue tommorow (1st March) . Alongside can i also start working on this issue and make a PR for the same tomorrow? Is it cool?

**Comment by Swati4star** [comment]
Have you committed the changes for your first issue in your forked repo? If you want, I can review those for now. And you can create PR (and I'll approve it)

**Comment by jddeep** [comment]
@Swati4star yes i did. You can do that.

**Comment by jddeep** [comment]
@Swati4star should I share you the link of my forked repo?

**Comment by jddeep** [comment]
@Swati4star I made a commit resolving this issue as well to my forked repo in the ` titlebarcount ` branch. Please review and approve that so I can take up another issue. Thanks.
